Spago docs --deps-only (#1080)
purescript/spago: README.md

$ spago docs --open
```

You can customize the output to other formats beyond html. Supported formats include ctags, etags, and markdown.
For example to generate ctags for use in your editor:
```console
$ spago docs --format ctags
```

> Sometimes you'd like to pull up docs for dependencies even when you have compilation errors in your project. This good use case for the --deps-only flag:

Reviewer: ```suggestion
Sometimes you'd like to pull up docs for dependencies even when you have compilation errors in your project - in such cases it's possible to use the --deps-only flag:
```

> Sometimes you'd like to pull up docs for dependencies even when you have compilation errors in your project. This good use case for the --deps-only flag:

Author: ```suggestion
Sometimes you'd like to pull up docs for dependencies even when you have compilation errors in your project. This is a good use case for the --deps-only flag:
```
